source/reduceBy.js

```diff
@@ -29,25 +29,22 @@ import _xreduceBy from './internal/_xreduceBy';
  * @see R.groupBy, R.reduce
  * @example
  *
- *      const reduceToNamesBy = R.reduceBy((acc, student) => acc.concat(student.name), []);
- *      const namesByGrade = reduceToNamesBy(function(student) {
- *        const score = student.score;
- *        return score < 65 ? 'F' :
- *               score < 70 ? 'D' :
- *               score < 80 ? 'C' :
- *               score < 90 ? 'B' : 'A';
- *      });
- *      const students = [{name: 'Lucy', score: 92},
- *                      {name: 'Drew', score: 85},
- *                      // ...
- *                      {name: 'Bart', score: 62}];
- *      namesByGrade(students);
- *      // {
- *      //   'A': ['Lucy'],
- *      //   'B': ['Drew']
- *      //   // ...,
- *      //   'F': ['Bart']
- *      // }
+ *      const groupNames = (acc, student) => acc.concat(student.name)
+ *      const toGrade = (student, score = student.score) =>
```

Comment:
TIL you could define parameters in terms of other parameters in JS. 

I had to run this in the REPL to make sure I knew what was going on, and you have a function that _appears_ to take two parameters even though the sig calls for one.

Would it be simpler to just do a basic destructuring: `const toGrade = ({score}) => ...`?

Reply:
Yes it would.  I wasn't thinking straight.  I was changing the existing function and wanted to make it a simple expression, and that version was the first thing that came to my mind.  I will to use simple destructuring.